gax/src/test/resources/com/google/api/gax/rpc/mtls/mtls_context_aware_metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cert_provider_command": [
+    "cat",
+    "src/test/resources/com/google/api/gax/rpc/mtls/mtlsCertAndKey.pem"
+  ],
+  "device_resource_ids": [
+    "123"
+  ]
+}
```

Comment:
minor: please add new line character here and in the other files where it is missing as well.

Reply:
done